### PR TITLE
DRUP-667 Display expired credentials as such

### PR DESCRIPTION
**Pull request by boobaa**
Related issue is #156.

### CODE REVIEW

File: apigee_edge.module [hunk 1]
```diff
@@ -992,11 +993,21 @@ function template_preprocess_app_credential(array &$variables) {
...
     }
     elseif ($def['value_type'] === 'status') {
-      $variables['primary_wrapper'][$property]['value'] = [
-        '#type' => 'status_property',
-        '#value' => $value,
-        '#indicator_status' => $value === AppCredentialInterface::STATUS_APPROVED ? StatusPropertyElement::INDICATOR_STATUS_OK : StatusPropertyElement::INDICATOR_STATUS_ERROR,
-      ];
+      $expires = $credential->getExpiresAt();
+      if ($expires && $expires->getTimestamp() < \Drupal::time()->getRequestTime()) {
+        $variables['primary_wrapper'][$property]['value'] = [
```
Comment (mxr576) [inline apigee_edge.module:998]: DRY. The render array is always the same, just the `#value` and the `#indicator_status` has to be calculated individually.

**Comment by mxr576** [comment]
@boobaa Please rebase your PR. 3f5a24b is not related to this PR.

**Comment by mxr576** [comment]
Also this PR does not address my comment on  https://github.com/apigee/apigee-edge-drupal/issues/156#issuecomment-471978381

**Comment by cnovak** [comment]
This has been fixed by #290

**Pull request closed without merging**